spec/domain/queries/find_aggregations_spec.rb

```diff
@@ -0,0 +1,83 @@
+RSpec.describe Queries::FindAggregations do
+  it 'A hash with a Key with each metric name' do
+    resut = Queries::FindAggregations.new.run
+    expect(resut).to be_a(Hash)
+    expect(resut.keys).to eq(Metric.find_all_names)
```

Comment:
If we are checking the keys - do we need to check its a Hash? (As this would fail anyway)